spec/features/submit_feedback_spec.rb

```diff
@@ -1,26 +1,50 @@
 require 'rails_helper'
 
 RSpec.feature 'Submit feedback', js: true do
-  before do skip 'Features specs not yet fixed' end
-
   include FeaturesHelper
 
-  scenario 'submitting feedback' do
+  before do
+    allow(PrisonVisits::Api.instance).to receive(:create_feedback)
```

Comment:
Why do you need to stub this api, if we are using vcr then it shouldn't be stubbed